blackbox/docs/release_notes/2.0.3.txt

```diff
@@ -0,0 +1,45 @@
+.. _version_2.0.3:
+
+=============
+Version 2.0.3
+=============
+
+Released on 2017/06/30.
+
+.. NOTE::
+
+   If you are upgrading a cluster, you must be running CrateDB
```

Comment:
This line goes over 79 chars if its not broken here, and I don't think theres a way to split up `:ref:`s over multiple lines :disappointed: